services/static-webserver/client/source/class/osparc/wrapper/Svg.js

```diff
@@ -156,6 +157,104 @@ qx.Class.define("osparc.wrapper.Svg", {
       return text;
     },
 
+    drawAnnotationNote: function(draw, x, y, destinataryName, note) {
+      const lines = note.split("\n");
+      const width = 200;
+      const minHeight = 120;
+      const titleHeight = 26;
+      const padding = 5;
+      const nLines = lines.length;
+      const height = Math.max(minHeight, titleHeight + nLines*18);
+      const trianSize = 25;
+      const yellow = "#FFFF01"; // do not make it pure yellow, svg will change the hex value to a "yellow" string
+      const orange = "#FFA500";
+      x = parseInt(x);
+      y = parseInt(y);
+
+      const gNote = draw.nested().move(x, y);
+      const rect = gNote.rect(width, height)
+        .fill(yellow)
+        .style({
+          cursor: "pointer"
+        });
+      rect.back();
+      gNote.add(rect);
+
+      const trianOrangeCtrls = [
+        [width-trianSize, height-trianSize],
+        [width-trianSize, height],
+        [width, height-trianSize]
+      ];
+      const trianOrange = gNote.polygon(trianOrangeCtrls.join())
+        .fill(orange)
+        .style({
+          cursor: "pointer"
+        })
+        .move(width-trianSize, height-trianSize);
+      trianOrange.back();
+      gNote.add(trianOrange);
+
+      const trianTransparentCtrls = [
+        [width-trianSize, height],
+        [width, height],
+        [width, height-trianSize]
+      ];
+      const colorManager = qx.theme.manager.Color.getInstance();
+      const trianTransparent = gNote.polygon(trianTransparentCtrls.join())
+        .fill(colorManager.resolve("background-main"))
+        .style({
+          cursor: "pointer"
+        })
+        .move(width-trianSize, height-trianSize);
+      trianTransparent.back();
+      colorManager.addListener("changeTheme", () => {
+        const bgColor = colorManager.resolve("background-main");
+        trianTransparent.fill(bgColor);
+      }, this);
+      gNote.add(trianTransparent);
+
+      const separator = gNote.line(0, 0, width-2*padding, 0)
+        .stroke({
+          width: 2,
+          color: orange
+        })
+        .move(padding, titleHeight);
+      separator.back();
+      gNote.add(separator);
+
+      const title = gNote.text(destinataryName)
```

Comment:
I would also put an `@` in front of the name to suggest that the message is addressed at that user. It will be consistent with what most social apps do.

Reply:
Only if we would have proper usernames. The recipient goes in the title, I think it's prominent enough.